src/UNL/UCBCN/Calendar.php

```diff
@@ -97,6 +98,10 @@ function links()
                      'uidlastupdated' => 'users:uid');
     }
 
+    public function manageHref() {
+        return "/manager/" . $this->shortname . "/manage/";
```

Comment:
I also wonder about this URL structure.  Would something like `/shortname/manage/` be better than `/manager/shortname/manage/`?